README.md

```diff
@@ -1,5 +1,130 @@
+
+Further, while the term annonymization is not defined within the GDPR, it is used by the European Data Protection Board ("EDPB") in further explaining the impact of the GDPR within the context of geolocation tracking. [3] The Guidelines state that
+
+> Anonymisation refers to the use of a set of techniques in order to remove the ability to link the data with an identified or identifiable natural person against any “reasonable” effort. This “reasonability test” must take into account both objective aspects (time, technical means) and contextual elements that may vary case by case (rarity of a phenomenon including population density, nature and volume of data). If the data fails to pass this test, then it has not been anonymised and therefore remains in the scope of the GDPR. ¶ 15
+
+Futher, these Guidelines outline three factors to determine the "robustmenss of anonymization": 
+
+> (i) singling-out (isolating an individual in a larger group based on the data); (ii) linkability (linking together two records concerning the same individual); and (iii) inference (deducing, with significant probability, unknown information about an individual). ¶ 16
+
+Based on this guidance, our project generated the following generalized defintion for "anonymization":
+
+> A process that removes the association between the identifying dataset and the data subject.
+
+#### Aggregation
+
+The concept of aggregated data bridges both the annonymity of information and the amount of information collected. In essence, it the combination of volume plus annonymity that allows aggregated data to be less prone to privacy concerns. The CCPA directly recognizes the concept of aggregation as follows:
+
+> information that relates to a group or category of consumers, from which individual consumer identities have been removed, that is not linked or reasonably linkable to any consumer or household, including via a device. “Aggregate consumer information” does not mean one or more individual consumer records that have been de­identified. [4]
+
+Additionally, the GDPR does recognize the concept of statistical research in Recital 162, stating:
+
+> Statistical purposes mean any operation of collection and the processing of personal data necessary for statistical surveys or for the production of statistical results. Those statistical results may further be used for different purposes, including a scientific research purpose. The statistical purpose implies that the result of processing for statistical purposes is not personal data, but aggregate data, and that this result or the personal data are not used in support of measures or decisions regarding any particular natural person.
+
+Based on these resources, our project generated the following generalized definition of "aggregation":
+
+> Information that relates to a group or category of individuals, from which individual identities have been removed, that is not linked or reasonably linkable to any individual.
+
```

Comment:
do we want to mention how many ?   2? 10? 100?   this is one of those "future" bullet points on our slide.  

Could add:  "aggregation might require all household energy usage data to be summed or averaged into groups of 15 or more household types, such as in California’s “15/15” rule (Harvey 2013)"

Harvey, S. J. (2013). Smart Meters, Smarter Regulation: Balancing Privacy and Innovation in the Electric Grid. UCLA L. Rev., 61, 2068.

Reply:
Below this we need a section for my use case metrics (from powerpoint/word doc) and proposed solution, I can always finish tomorrow after recording if we run out of time.  And relate them to the above (check powerpoint)